season_pass/api/user.py

```diff
@@ -303,6 +303,14 @@ def claim_reward(request: ClaimRequestSchema, sess=Depends(session)):
             f"No activity recorded for season {target_pass.id} for avatar {request.avatar_addr}"
         )
 
+    inprogress_claim_count = sess.scalar(select(func.count()).where(
+        Claim.reward_list != [],
+        or_(Claim.tx_status == TxStatus.STAGED, Claim.tx_status == TxStatus.INVALID)
+    ))
+
+    if inprogress_claim_count > 10:
```

Comment:
I think 10 is too tight. Event organic users can generate hundreds of claims easily

Reply:
50 ~ 100